.github/scripts/tests/README.md

```diff
@@ -58,7 +61,34 @@ Replace `username` with your quay user
 export REGISTRY_ADDRESS=quay.io/username
 ```
 
-# Run the test
+## Run the test
 ```bash
 sh .github/scripts/tests/tests.sh --kind
 ```
+
+# Debug a test using GoLand
+Let's say you wish to debug the `Should create a Pipeline Run` test.
+The first step is right click inside the method content and select the menu
+`Run 'TestIntegrationTestSuite'`. It will fail because you need to fill some parameters.
+Edit the configuration for `TestIntegrationTestSuite/TestPipelineSuccessfulRun/Should_create_a_Pipeline_Run in github.com/opendatahub-io/data-science-pipelines-operator/tests`
```

Comment:
I can't find this configuration. I guess this is generated when you run for the first time?

Reply:
Yes.